packages/frame/src/FragmentFrame.ts

```diff
@@ -25,6 +25,10 @@ export class FragmentFrame<Name extends string = string> implements IteratableFr
     return this.origin.getCursor(column);
   }
 
+  public groupBy(columns: Array<Name | ColumnCursor<Name>>) {
```

Comment:
```suggestion
  public groupBy(columns: Array<Name> | Array<ColumnCursor<Name>>) {
```

one or another, but not both in the same array. Matter of taste I guess